.pre-commit-config.yaml

```diff
@@ -1,29 +1,35 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v3.1.0
     hooks:
     -   id: check-yaml
+
+-   repo: https://github.com/asottile/seed-isort-config
+    rev: v2.2.0
+    hooks:
+    -   id: seed-isort-config
+
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v4.3.21
+    hooks:
+      - id: isort
+
 -   repo: https://github.com/psf/black
-    rev: 19.3b0
+    rev: 19.10b0
     hooks:
       - id: black
-        name: fmt
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
+        name: black
+
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.3
     hooks:
     -   id: flake8
+
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.720
+    rev: v0.780
```

Comment:
I've been avoiding bumping the `mypy` version because I knew it'd cause a lot of new failures..  looking at the CI thought it's not as bad as I expected, only 9 errors.  Mind handling that as well?

Reply:
reverted back to v0.720 on the latest commit